saba/tests/test_main.py

```diff
@@ -343,3 +344,25 @@ def test_check_fit(self, nbins=100):
             x_med = (xx[med_ind] + xx[med_ind + 1]) / 2.0
 
             assert_allclose(self.params[nn], x_med, atol=0.1)
+
+
+
+class TestSherpaQuantiesFitter(object):
+
+    def setup_class(self):
+        self.x = np.linspace(1, 5, 30) * u.micron
+        self.y = np.exp(-0.5 * (self.x - 2.5 * u.micron)**2 / (200 * u.nm)**2) * u.mJy
+        self.model_micron = Gaussian1D(mean=3 * u.micron, stddev=1 * u.micron, amplitude=1 * u.Jy)
+        self.model_thz =  Gaussian1D(mean=110 * u.THz, stddev=10 * u.THz, amplitude=1 * u.Jy)
+        self.fitter = SherpaFitter()
+
+    def test_basic(self):
+        self.fitter(self.model_micron, self.x, self.y)
+
```

Comment:
Just trying to check that no exceptions are thrown at the moment.
I was planning extending the tests over this weekend to test the values make sense.